quadfeather/tiler.py

```diff
@@ -153,30 +153,14 @@ def refine_schema(schema: pa.Schema) -> Dict[str, pa.DataType]:
     fields = {}
     seen_ix = False
     for el in schema:
-        if isinstance(el.type, pa.DictionaryType) and pa.types.is_string(
-            el.type.value_type
-        ):
-            t = pa.dictionary(pa.int16(), pa.utf8())
-            fields[el.name] = t
-        #            el = pa.field(el.name, t)
-        elif pa.types.is_float64(el.type) or pa.types.is_float32(el.type):
-            fields[el.name] = pa.float32()
-        elif el.name == "ix":
+        if el.name == "ix":
             fields[el.name] = pa.uint64()
             seen_ix = True
-        elif pa.types.is_integer(el.type):
-            # Integers become float32
-            fields[el.name] = pa.float32()
-        elif pa.types.is_large_string(el.type) or pa.types.is_string(el.type):
-            fields[el.name] = pa.string()
-        elif pa.types.is_boolean(el.type):
-            fields[el.name] = pa.float32()
-        elif pa.types.is_date32(el.type):
-            fields[el.name] = pa.date32()
-        elif pa.types.is_temporal(el.type):
+        elif pa.types.is_date(el.type):
+            # Deepscatter can't handle all the different date types.
             fields[el.name] = pa.timestamp("ms")
```

Comment:
Do we need to be careful here? Is converting a date object to timestamp safe?
`pa.scalar(date.today(), type=pa.timestamp('ms'))` something like this will throw an error

Reply:
good question but I added a test [here](https://app.graphite.dev/github/pr/bmschmidt/quadfeather/15/add-a-test-of-datetime-timestamp-tests) and it seems to work fine. Difference from your example is that we're not converting the native python type, but just from one arrow type to another.